buildConfig.fsx

```diff
@@ -30,20 +30,13 @@ open AssemblyInfoFile
 if isMono then
     monoArguments <- "--runtime=v4.0 --debug"
 
-let projectName_roslyn = "RazorEngine.Roslyn"
-let projectSummary_roslyn = "Roslyn extensions for RazorEngine."
-let projectDescription_roslyn = "RazorEngine.Roslyn - Roslyn support for RazorEngine."
 // !!!!!!!!!!!!!!!!!!!
 // UPDATE RELEASE NOTES AS WELL! (set 'nugetkey' environment variable to push directly.)
 // !!!!!!!!!!!!!!!!!!!
 let version_razor4 = "4.5.0-rc1"
 let version_roslyn = "3.6.1-rc1"
 let version_roslyn_razor4 = "4.1.1-rc1"
```

Comment:
Maybe it's time to get over it and drop net40 support to get rid of all this mess for the 4.X release?